pkg/kubelet/apis/config/scheme/testdata/KubeletConfiguration/after/v1beta1.yaml

```diff
@@ -36,7 +37,6 @@ evictionHard:
   imagefs.available: 15%
   memory.available: 100Mi
   nodefs.available: 10%
-  nodefs.inodesFree: 5%
```

Comment:
Not sure why this got removed; I need to check.
I had consistent test failures on my mac.